arrocke/gloss-translation
# feat: add tab group to sidebar

## What has changed


Added tab group to the sidebar.
## Connected Issues



#303 

## QA Steps



1. Open the sidebar on the translation page
2. Try the different tabs
3. Check that scrolling works correctly for long lexicon entries

## Post-Deployment
